GitUI/UserControls/RevisionGrid.cs

```diff
@@ -1886,19 +1886,11 @@ private void RevisionsCellFormatting(object sender, DataGridViewCellFormattingEv
             else if (AppSettings.ShowIndicatorForMultilineMessage && columnIndex == isMsgMultilineColIndex)
             {
                 if (revision.Body == null && !revision.IsArtificial())
-                {
-                    var moduleRef = Module;
-                    ThreadPool.QueueUserWorkItem(o => LoadIsMultilineMessageInfo(revision, columnIndex, e.RowIndex, Revisions.RowCount, moduleRef));
-                }
+                    Task.Run(async () => await LoadIsMultilineMessageInfoAsync(revision, columnIndex, e.RowIndex, Revisions.RowCount));
```

Comment:
❗️ Asynchronous operation not captured in the lifetime of the enclosing operation must be explicitly tracked

❗️ Missing `ConfigureAwait(false)`